README.md

```diff
@@ -55,8 +55,11 @@ The most convenient way to configure delta is with a `[delta]` section in `~/.gi
 <sub>
 
 ```gitconfig
-[core]
-    pager = delta
+[pager]
+    diff = delta
+    log = delta
+    reflog = delta
+    show = delta
```

Comment:
Am I missing any?

Reply:
Sorry for the late response @dandavison - maybe you could add `stash` ?